src/Common/FiberStack.h

```diff
@@ -33,10 +33,12 @@ class FiberStack
     /// Current value is just enough for all tests in our CI. It's not selected in some special
     /// way. We will have 40 pages with 4KB page size.
     static constexpr size_t default_stack_size = 192 * 1024; /// 64KB was not enough for tests
+    static constexpr size_t default_page_size = 4096;
 
     explicit FiberStack(size_t stack_size_ = default_stack_size) : stack_size(stack_size_)
     {
-        page_size = ::sysconf(_SC_PAGESIZE);
```

Comment:
By the way, have you seen this failure in a some real env? Or it is just some theoretical thing?

Reply:
theoretical but has happened and there is assertion for it, not sure what the consequences would be if this were to occur on a system running clickhouse. I would likely need to be caused by bug or misconfiguration.
for reference:
https://bugs.launchpad.net/ubuntu/+source/eglibc/+bug/672352

Reply:
>theoretical but has happened and there is assertion for it

Happened in your env or in the linked issue?
But I'm asking just out of curiosity and this patch anyway makes sense.

Also take a look at https://github.com/ClickHouse/ClickHouse/blob/f2ebeb6d3f3480d6344d4d380031e47e00fb4c71/base/base/getPageSize.cpp#L7

I think it is better to fix it instead, and use `getPageSize` here.

Reply:
theoretical, as we have never seen it happen.